modules/react/button/index.ts

```diff
@@ -9,6 +9,11 @@ export {
 export {ToolbarIconButton, ToolbarIconButtonProps} from './lib/ToolbarIconButton';
 export {ToolbarDropdownButton, ToolbarDropdownButtonProps} from './lib/ToolbarDropdownButton';
 export {Hyperlink, HyperlinkProps} from './lib/Hyperlink';
-export {BaseButton, ButtonContainerProps} from './lib/BaseButton';
+export {
+  BaseButton,
+  ButtonContainerProps,
```

Comment:
So as you created `BaseButtonProps` can you replace `ButtonContainerProps` by `BaseButtonProps` in export?

Reply:
yep, makes sense to me!